www/public/index.html

```diff
@@ -10,6 +10,7 @@
         <meta content="minimum-scale=1, initial-scale=1, width=device-width" />
         <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
         <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+        <script type="text/javascript" src="https://documentcloud.adobe.com/view-sdk/main.js"></script>
```

Comment:
No npm package I'm assuming?

Reply:
Not that I could find for the browser :/

Reply:
Yea seems like this is the norm (or at least this is what they gave in [the samples](https://github.com/adobe/pdf-embed-api-samples/tree/master/More%20Samples/React%20Samples))